flytekit/core/context_manager.py

```diff
@@ -89,6 +89,8 @@ class Builder(object):
         execution_date: typing.Optional[datetime] = None
         logging: Optional[_logging.Logger] = None
         task_id: typing.Optional[_identifier.Identifier] = None
+        rendered_decks: Optional[List[str]] = None
+        output_metadata_prefix: Optional[str] = None
```

Comment:
For the current implementation, in https://github.com/flyteorg/flytekit/blob/d504753cd74ff1b1ad5f399d5e5c3573dfbdad0d/flytekit/bin/entrypoint.py#L177, a new param is passed in, so `output_metadata_prefix ` has to be included otherwise the field will be empty and cause error.